Kickstarter-iOS/Views/Controllers/PledgeAmountViewControllerTests.swift

```diff
@@ -77,13 +77,13 @@ final class PledgeAmountViewControllerTests: TestCase {
     }
   }
 
-  func testView_StepperIncrementButtonDisabled_WhenStepperValueSetToMaximum() {
+  func testView_StepperIncrementButtonDisabled_WhenStepperValueSetToMaximumStepperValue() {
```

Comment:
Renamed couple tests for better understanding of what min/max are we referring to (since there is stepper min/max value, pledge min/max value, etc.)